sentinel/pkg/checker/dal/app.go

```diff
@@ -110,6 +163,98 @@ func checkDal(endpoint string, key string, alarmCount map[string]int) error {
 	return nil
 }
 
+func checkDalWs(ctx context.Context) {
+	log.Info().Msg("checking msg delays")
```

Comment:
minor suggestion, how about downgrading it to `debug`? doesn't seem to be an important information